php-8.2.yaml

```diff
@@ -7,7 +7,7 @@ package:
     - license: PHP-3.01
   dependencies:
     provides:
-      - php=${{package.version}}-r${{package.epoch}}}
+      - php=${{package.version}}-r${{package.epoch}}
```

Comment:
FYI: You can replace this with ${{package.full-version}}

Reply:
thanks! i just realized I forgot to bump the epoch, will do it now!